doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -652,6 +652,26 @@ The following changes are recommended for your application to work optimally aft
 
 * For LwM2M applications, replace the :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_DTLS_CID` Kconfig option with :kconfig:option:`CONFIG_LWM2M_DTLS_CID`.
 
+* For Mesh applications, the Sensor API has been re-written, and the previous Sensor API is to be depreciated.
+  A new Sensor API has been created that should be used for new applications and applications that will be continuously updated.
+  How the new Sensor API is working is shown in the :ref:`bluetooth_mesh_sensor_client`, :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_light_lc` samples, as they all have been updated to use the new Sensor API.
+  It is important to use the new Sensor API as it has a better precision than the deprecated Sensor API.
```

Comment:
I'd remove this entire sentence. It's incorrect as it is, it could be rewritten to "It is important to use the new Sensor API as it can provide better precision in some use cases", but then it doesn't have much value ("why is it then important if it doesn't provide any value in my use case?"). The changelog already mentions increased precision, those who don't need that for their use case won't use that as an argument for using the new API.